src/livekit/BlurBackgroundTransformer.ts

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2024 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only
+Please see LICENSE in the repository root for full details.
+*/
+
+import {
+  BackgroundTransformer,
+  VideoTransformer,
+  VideoTransformerInitOptions,
+} from "@livekit/track-processors";
+import { ImageSegmenter } from "@mediapipe/tasks-vision";
+
+interface WasmFileset {
+  /** The path to the Wasm loader script. */
+  wasmLoaderPath: string;
+  /** The path to the Wasm binary. */
+  wasmBinaryPath: string;
+}
+
+// n.b. this only includes the SIMD versions of the WASM files which have good support:
+// https://caniuse.com/?search=simd
+const wasmFileset: WasmFileset = {
+  wasmLoaderPath: new URL(
+    "../../node_modules/@mediapipe/tasks-vision/wasm/vision_wasm_internal.js",
```

Comment:
I think it would be better to use

```
import wasmLoaderPath from "@mediapipe/tasks-vision/wasm/vision_wasm_internal.js?url";
```

Reply:
I did try that, but I ended up with a vite error and so did this approach instead. 🤷